GitUI/CommandsDialogs/SettingsDialog/Pages/SshSettingsPage.cs

```diff
@@ -27,15 +26,13 @@ public static SettingsPageReference GetPageReference()
         {
             return new SettingsPageReferenceByType(typeof(SshSettingsPage));
         }
-        private ConfigFileSettings GlobalConfigFileSettings { get { return CommonLogic.ConfigFileSettingsSet.GlobalSettings; } }
 
         protected override void SettingsToPage()
         {
             PlinkPath.Text = AppSettings.Plink;
             PuttygenPath.Text = AppSettings.Puttygen;
             PageantPath.Text = AppSettings.Pageant;
             AutostartPageant.Checked = AppSettings.AutoStartPageant;
-            GitCredPath.Text = GlobalConfigFileSettings.GetValue("credential.helper");
```

Comment:
why wouldn't https work without it? and i don't quite understand, i removed SuggestGitCred...